Cargo.toml

```diff
@@ -39,6 +39,7 @@ sha3 = "^0.10"
 hex-literal = "^0.3"
 serde_json = "1.0.61"
 derive_more = "0.99.17"
+primitive-types = "0.11.1"
```

Comment:
I have slight concern for type compatibility.

CAPE uses ethers-rs, which internally use [`ethabi`](https://github.com/rust-ethereum/ethabi) for its `U256`, see [here](https://github.com/gakonst/ethers-rs/blob/master/ethers-core/src/types/u256.rs#L1)

maybe we should switch to that?

Reply:
It's the same type. But yeah, probably better to depend on it the same way ethers does

Reply:
yeah done in [3524452](https://github.com/EspressoSystems/cap/pull/49/commits/3524452c6b7ecb79d0e1d08c8a7ca2aa74c157bc), could you pls approve @jbearer ?